components/icu4x_ecma402/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "icu4x_ecma402"
+version = "0.0.1"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "../../LICENSE"
+categories = ["internationalization"]
+include = [
+    "src/**/*",
+    "Cargo.toml",
+    "README.md"
+]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ecma402_traits = { version = "0.2.0" }
+icu-data-provider = { path = "../data-provider" }
+icu-locale = { path = "../locale" }
+icu-plurals = { path = "../plurals" }
```

Comment:
Consider using `icu` meta-package for it. Since this is a "meta" package for all ecma402 traits. Not a blocker, up to code author.

Reply:
Done.